src/OutsideClickHandler.jsx

```diff
@@ -16,7 +15,8 @@ const propTypes = forbidExtraProps({
   onOutsideClick: PropTypes.func.isRequired,
   disabled: PropTypes.bool,
   useCapture: PropTypes.bool,
-  display: PropTypes.oneOf(objectValues(DISPLAY)),
+  style: PropTypes.style,
+  className: PropTypes.string,
```

Comment:
We have very intentionally disallowed this antipattern; please see https://brigade.engineering/don-t-pass-css-classes-between-components-e9f7ab192785?gi=d97e4a1f93eb